packages/openactive-broker-microservice/src/twoPhaseListeners/api.js

```diff
@@ -61,14 +55,14 @@ function getOrderListenerApi(req, res) {
  * @returns {boolean} `true` if it was found. `false` if no listener was found.
  */
 function doPendOrRespondToGetListenerRequest(res, listenersMap, listenerId) {
-  if (!state.listeners.byOrderUuid.has(listenerId)) {
+  if (!listenersMap.has(listenerId)) {
```

Comment:
Oops! I think this didn't show up as a bug before because there don't actually seem to be any tests that use the Opportunity Feed Listener FlowStage